packages/framework/tinylicious-client/src/interfaces.ts

```diff
@@ -44,6 +44,13 @@ export interface TinyliciousConnectionConfig {
      * @defaultValue {@link @fluidframework/tinylicious-driver#defaultTinyliciousEndpoint}
      */
     domain?: string;
+
+    /**
+     * Optional. Override of tokenProvider.
```

Comment:
Please add description and explain what happens when default not offered. I would not assume that consumers of this class know what internal behavior we are overriding.